config/mpc_params.yaml

```diff
@@ -0,0 +1,10 @@
+save_predictions_and_refs: False
+dt: 0.01
+rate: 100
+horizon_size: 70
+armature: [0.05, 0.05, 0.05, 0.05, 0.05, 0.05, 0.05]
+gripper_weight: 10000
+state_weight: 10
+control_weight: 0.001
+use_constraints: False
+ros_namespace: '/ctrl_mpc_linearized'
```

Comment:
this shouldn't be needed as ros automatically maps the namespaces in the topic and node names